package-lock.json

```diff
@@ -172,6 +172,12 @@
             "source-map": "^0.5.0"
           },
           "dependencies": {
+            "lodash": {
+              "version": "4.17.21",
+              "resolved": "https://repository.porscheinformatik.com/repository/npm-public/lodash/-/lodash-4.17.21.tgz",
```

Comment:
this is a public repository, so no internal porsche links should be used.
if you don't change anything in package.json, just use "npm ci" to install dependencies

Reply:
done